contentcuration/contentcuration/frontend/RecommendedResourceCard/components/RecommendedResourceCard.vue

```diff
@@ -3,6 +3,7 @@
   <KCard
     :title="node.title"
     :headingLevel="2"
+    class="test"
```

Comment:
Is this necessary?

Reply:
This one just crept through. will update shortly. Thanks